site/src/components/WorkspaceScheduleForm/WorkspaceScheduleForm.tsx

```diff
@@ -154,21 +178,13 @@ export const validationSchema = Yup.object({
 export const WorkspaceScheduleForm: FC<WorkspaceScheduleFormProps> = ({
   fieldErrors,
   initialValues = {
-    sunday: false,
-    monday: true,
-    tuesday: true,
-    wednesday: true,
-    thursday: true,
-    friday: true,
-    saturday: false,
-
-    startTime: "09:30",
-    timezone: "",
-    ttl: 5,
+    ...WorkspaceScheduleFormInitialValues,
```

Comment:
**Suggestion**: We don't need to spread here (spread is for copying, or merging objects). It's a little simpler if we just:

```ts
initialValues = WorkspaceScheduleFormInitialValues,
```